tasks/bake.js

```diff
@@ -384,6 +384,25 @@ module.exports = function( grunt ) {
 			return null;
 		}
 
+		// Handle _bake attributes in inline arguments
+
+		function validateExtraBake( inlineValues ) {
```

Comment:
What do you think, should we rename this `validateBake` since we call the other one `validate<nameOfAttribute>` such as [`validateIf`](https://github.com/MathiasPaumgarten/grunt-bake/blob/master/tasks/bake.js#L236) and [`validateForEach`](https://github.com/MathiasPaumgarten/grunt-bake/blob/master/tasks/bake.js#L313)?